mule-artifact-it/mule-packager-it/src/test/resources/expected-classloader-model-project/META-INF/mule-artifact/classloader-model.json

```diff
@@ -14,7 +14,8 @@
         "version": "1.0.0-SNAPSHOT",
         "type": "jar"
       },
-      "uri": "repository/group/id/a/artifact-id-a/1.0.0-SNAPSHOT/artifact-id-a-1.0.0-SNAPSHOT.jar"
+      "uri": "repository/group/id/a/artifact-id-a/1.0.0-SNAPSHOT/artifact-id-a-1.0.0-SNAPSHOT.jar",
+      "isShared": "false"
```

Comment:
Maybe isShared=false should be the default by convention and you don't start adding isShared:false to every place and rather isShared:true to those dependencies that are shared.